runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/codestart.yml

```diff
@@ -0,0 +1,20 @@
+name: quarkus-hilla-maven-codestart
+ref: quarkus-hilla-maven-starter
```

Comment:
From where comes this ref?

Reply:
From the docs, but it is wrong. It should be the extension id.
I'll fix it
https://quarkus.io/guides/extension-codestart#codestart-yml

Reply:
done